dpu-cni/pkgs/sriov/sriov.go

```diff
@@ -371,10 +379,11 @@ func (sm *sriovManager) CmdAdd(req *cnitypes.PodRequest) (*current.Result, error
 	}
 	defer netns.Close()
 
-	err = sm.FillOriginalVfInfo(netConf)
-	if err != nil {
-		return nil, fmt.Errorf("failed to get original vf information: %v", err)
-	}
+	// TODO: Some vendors have issues with Host VLANs, thus disable the feature for now
```

Comment:
Change to "// TODO: Some vendors have issues with Netlink, thus disable the feature for now"